### PR TITLE
feat(FEC-9142): getting bumper from provider by entry Id (OVP)

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

parse bumper from playback context - OVP

Solves FEC-9142

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/k-provider/ovp/provider.js [hunk 1]
```diff
@@ -117,6 +119,22 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
           this._verifyHasSources(mediaConfig.sources);
         }
       }
+      if (data.has(OVPBumperEntryLoader.id)) {
+        const bumperLoader = data.get(OVPBumperEntryLoader.id);
+        if (bumperLoader && bumperLoader.response) {
+          if (OVPProviderParser.hasBlockAction(bumperLoader.response)) {
+            new Error(Error.Severity.CRITICAL, Error.Category.SERVICE, Error.Code.BLOCK_ACTION, {
```
Comment (OrenMe) [inline src/k-provider/ovp/provider.js]: No need to fail media if bumper has AC

File: src/k-provider/ovp/provider.js [hunk 2]
```diff
@@ -47,6 +48,7 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
         }
         const redirectFromEntryId = this._getEntryRedirectFilter(mediaInfo);
         this._dataLoader.add(OVPMediaEntryLoader, {entryId, ks, redirectFromEntryId});
+        this._dataLoader.add(OVPBumperEntryLoader, {entryId: mediaInfo.bumperId, ks, redirectFromEntryId});
```
Comment (OrenMe) [inline src/k-provider/ovp/provider.js]: we always add it now?

File: src/k-provider/ovp/loaders/bumper-entry-loader.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+   * @param {Object} params Requests parameters
+   * @returns {RequestBuilder} The request builder
+   * @static
+   */
+  buildRequests(params: Object): Array<RequestBuilder> {
+    const config = OVPConfiguration.get();
+    const requests: Array<RequestBuilder> = [];
+    const listRequest = OVPBaseEntryService.list(config.serviceUrl, params.ks, params.entryId, params.redirectFromEntryId);
+    const playbackContextRequest = OVPBaseEntryService.getPlaybackContext(config.serviceUrl, params.ks, params.entryId);
+    listRequest.mandatory = playbackContextRequest.mandatory = false;
```
Comment (OrenMe) [inline src/k-provider/ovp/loaders/bumper-entry-loader.js]: seems like this should get to the provider level and not be added at all if no provider ID was set

File: src/k-provider/ovp/loaders/bumper-entry-loader.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+   * Builds loader requests
+   * @function
+   * @param {Object} params Requests parameters
+   * @returns {RequestBuilder} The request builder
+   * @static
+   */
+  buildRequests(params: Object): Array<RequestBuilder> {
+    const config = OVPConfiguration.get();
+    const requests: Array<RequestBuilder> = [];
+    const listRequest = OVPBaseEntryService.list(config.serviceUrl, params.ks, params.entryId, params.redirectFromEntryId);
```
Comment (OrenMe) [inline src/k-provider/ovp/loaders/bumper-entry-loader.js]: why do we need the base entry? playbackContext is sufficient for the bumper.

File: src/k-provider/ovp/loaders/bumper-entry-loader.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  set requests(requests: Array<RequestBuilder>) {
+    this._requests = requests;
+  }
+
+  get requests(): Array<RequestBuilder> {
+    return this._requests;
+  }
+
+  set response(response: any) {
+    if (response[0].hasError) {
```
Comment (OrenMe) [inline src/k-provider/ovp/loaders/bumper-entry-loader.js]: handling error here seems to break or way of handling this in other loaders, and especially creating an error here.

File: src/k-provider/ovp/provider.js (hunk 2 above)
Comment (yairans) [reply src/k-provider/ovp/provider.js]: `OVPBumperEntryLoader` is invalid when no id so it won't be added actually

File: src/k-provider/ovp/loaders/bumper-entry-loader.js (hunk 3 above)
Comment (yairans) [reply src/k-provider/ovp/loaders/bumper-entry-loader.js]: `mandetory` means this request should not fail the entire multi request if it gets an error from the BE

**Review by OrenMe — APPROVED** [review]

**Pull request merged**